dist/clouduploader.rb

```diff
@@ -5,98 +5,83 @@
 require 'ostruct'
 require 'open3'
 
-start = Time.now
-THIRTY_MINUTES = 1800
-HOME = '/etc/obs/cloudupload'.freeze
-ENV['HOME'] = "/etc/obs/cloudupload"
-ENV['PYTHONUNBUFFERED'] = '1'
-STDOUT.sync = true
-
-if ARGV.length != 6
-  raise 'Wrong number of arguments, please provide: user platform upload_file targetdata filename result_path'
-end
-
-platform = ARGV[1]
-image_path = ARGV[2]
-data_path = ARGV[3]
-filename = ARGV[4]
-result_path = ARGV[5]
-data = JSON.parse(File.read(data_path))
+module CloudUploader
```

Comment:
The src/backend/build change does not belong to this commit (nor the PR I believe)